model/src/main/kotlin/OrtError.kt

```diff
@@ -34,7 +34,7 @@ import java.time.Instant
 /**
  * An error that occured while executing ORT.
  */
-data class Error(
+data class OrtError(
```

Comment:
Would it make sense to not affect the serialization at all to not increase file size and keep backwards compatible by
using error as type name, e.g. '@JsonTypeName("Error")' ?

Reply:
We don't write any type names during serialization so the output does not change. That's a feature you usually only need when you use inheritance (see https://github.com/FasterXML/jackson-docs/wiki/JacksonPolymorphicDeserialization).

Reply:
oh great, invalid comment from my side then.